### ISSUE TITLE
Nest 3 : Problem of reproducibility and extrem firing rate of neurons

### ISSUE THREAD

**Issue by lionelkusch** [nest/nest-simulator#1409]
I test Nest 3 and I get strange behavior of the firing rate.
I try to reduce my code in order to isolate the problems.

I get the following result with the version 2.18.0 : 

![test_nest_2_18](https://user-images.githubusercontent.com/17182418/73062852-a824e800-3e9d-11ea-8706-a329340c29ef.png)
 
When I use the commit fbebdc0aefa4a06e09d6647f19c695bf8071d640 of Nest 3, I get the following result :

![test_nest_3](https://user-images.githubusercontent.com/17182418/73062807-89265600-3e9d-11ea-8b39-e19e60e2dfc0.png)

We can see the result are different for each run.
Moreover, some firing rate are to high for the parameters.

This is the code for reproduce this figure:

[problem_nest.txt](https://github.com/nest/nest-simulator/files/4107914/problem_nest.txt)

**Comment by lionelkusch** [comment]
I am sorry for my late answer.
The fix of this bug doesn't solve my problems.
I get the next result with the commit c8f2a61943e3ed3cdfb1704dff86ad931a96c6a9 :
![test_nest_3_v1](https://user-images.githubusercontent.com/17182418/76415681-e87eed80-6399-11ea-81f9-a8cc89ea6e31.png)

This is the output for the previous python file:
[log.txt](https://github.com/nest/nest-simulator/files/4318166/log.txt)

**Comment by hakonsbm** [comment]
@lionelkusch The fix is in the PR #1443, which is actually not merged into master just yet. To see if the fix works for you, you need to check out and compile @stinebuu's branch:

https://github.com/stinebuu/nest-simulator/tree/bld_issue1409_thread_devices

If you find that the fix still doesn't work, please make a comment in #1443.

**Comment by lionelkusch** [comment]
@hakonsbm : The PR #1443  fix this bug. 

Thank you for your work and sorry for my previous comment. I was thinking the pull request was already merge with master.

**Comment by heplesser** [comment]
Reopening, since the PR fixing this issue is not yet merged. The issue will be closed automatically once the PR is merged.
